pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package compatibility
+package testing
```

Comment:
I added a `policy_test.go` into this folder, so this folder is not about compatibility-only testing any more.